### PR TITLE
fix: avoid TypeError when using sorted search

### PR DESCRIPTION
**Pull request by tswast**
Based on #64

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-db-dtypes-pandas/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Towards #28 🦕

### CODE REVIEW

**Comment by tswast** [comment]
Re: "Do not merge" I meant for this to be a Draft until #64 can be merged.

**Review by stephaniewang526 — APPROVED** [review]

**Pull request merged**
